docker/jenkins/Makefile

```diff
@@ -2,13 +2,22 @@ BASE_FILE=docker-compose.yml
 OVERRIDE_FILE=docker-compose.override.yml
 SERVICE=ci
 SETUP_SCRIPT=../nvidia/setup_nvidia_docker_compose.py
+ENV_VARIABLES=CUDA_VERSION GPU_ARCH
 
 all: setup_nvidia_docker
 
 setup_nvidia_docker: ${OVERRIDE_FILE}
 
-${OVERRIDE_FILE}: ${BASE_FILE} ${SETUP_SCRIPT} Makefile
+${OVERRIDE_FILE}: ${BASE_FILE} ${SETUP_SCRIPT} Makefile ${ENV_VARIABLES}
```

Comment:
and those new variables get passed me make is called

Reply:
Not exactly.  We write stamp files that save the values of these env variables at the time the setup script was last called and add a dependency on these files.